source/features/index.tsx

```diff
@@ -85,11 +93,20 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 	}
 
 	if (document.body.classList.contains('logged-out')) {
-		console.warn('%cRefined GitHub%c is only expected to work when you’re logged in to GitHub.', 'font-weight: bold', '');
+		console.warn('Refined GitHub is only expected to work when you’re logged in to GitHub. Errors will not be shown.');
+		features.error = () => {/* No logging */};
+		logError = () => {/* No logging */};
 	}
```

Comment:
## No more errors for logged-out users

Only the warning will be shown

<img width="656" alt="Screen Shot 8" src="https://user-images.githubusercontent.com/1402241/116375999-04fa1580-a83a-11eb-8f23-89674a73b4b9.png">